src/content/docs/blog/caching-update.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: The Caching Update
+description: We've been working hard to make your building experience smoother!
+date: 2024-04-2X
+authors:
+    - name: Gerald (Jerry) Pinder
+      img: https://avatars.githubusercontent.com/u/4626052
+      github: gmpinder
+---
+
+Hello! I'm Jerry and I'm the creator and maintainer of the Rust-based `bluebuild` CLI tool. I started creating this tool as a way to extend the work that [Xyny](https://github.com/xynydev) did with the recipe standard, but make it so that I didn't have to manage the `Containerfile` directly.
```

Comment:
```suggestion
Hello! I'm Jerry and I'm the creator and maintainer of the Rust-based `bluebuild` CLI tool. I started creating this tool as a way to extend the work that [XYNY](https://github.com/xynydev) did with the recipe standard, but make it so that I didn't have to manage the `Containerfile` directly.
```